src/main/java/com/team766/controllers/CanSparkMaxSmartMotionRotationalPIDController.java

```diff
@@ -0,0 +1,34 @@
+import java.*;
+
+public class CanSparkMaxSmartMotionRotationalPIDController extends ExtendableCanSparkMaxSmartMotionPIDController {
+	
+	private final double degreesToEncoderUnitsRatio;
+	private final double antiGravK;
```

Comment:
it won't work to make this `final`, because its value might change

Reply:
it shouldn't, and now it especially won't.

Reply:
`setAntigravConstant` tries to change this value

Reply:
sorry, i thought we were talking about something different. You're totally right, and its fixed.